src/panels/config/logs/dialog-download-logs.ts

```diff
@@ -126,6 +125,13 @@ class DownloadLogsDialog extends LitElement {
     this.closeDialog();
   }
 
+  private _getDownloadUrl() {
+    if (this._dialogParams!.boot === 0) {
+      return getHassioLogDownloadLinesUrl;
+    }
+    return getHassioLogBootDownloadLinesUrl;
+  }
+
```

Comment:
These methods that return a function are a bit weird. I would expect `_getDownloadUrl` to return a url, not a function. It would be better to make this a getter, so it can be called directly without the extra `()` or just rename it to `_getDownloadUrlFn` or something similar.
Same for `_fetchLogs` in `error-log-card`